react-accordion/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+import { TopicCard } from './TopicCard.tsx';
+
+type Topic = {
+  id: number;
+  title: string;
+  content: string;
+};
+
+type Props = {
+  topics: Topic[];
+};
+
+export function Accordion({ topics }: Props) {
+  const [currentTopic, setCurrentTopic] = useState('');
```

Comment:
If you have 2 topics with the same text then this won't work. Topics were given an ID to address this. You should use the ID instead of the text.